goaviatrix/tags.go

```diff
@@ -122,7 +161,7 @@ func (c *Client) DeleteTags(tags *Tags) error {
 	return nil
 }
 
-func (c *Client) AzureUpdateTags(tags *Tags) error {
+func (c *Client) UpdateTags(tags *Tags) error {
 	tags.CID = c.CID
```

Comment:
I haven't tried it my self, but I remember `update_resource_tags` sends the `action=replace` to the backend cli. AWS and GCP don't support `replace`.

Reply:
AWS now supports replace, I checked the code in the backend and tested it.  I haven't tried with GCP, I will check.

Reply:
The API does not support GCP tags.